src/test/scala/com/microsoft/hyperspace/index/IndexManagerTests.scala

```diff
@@ -67,7 +67,7 @@ class IndexManagerTests extends HyperspaceSuite with SQLHelper {
             StructType(Seq(StructField("RGUID", StringType), StructField("Date", StringType)))
           if (enableLineage) {
             expectedSchema = expectedSchema.add(
-              StructField(IndexConstants.DATA_FILE_NAME_COLUMN, StringType, false))
+              StructField(IndexConstants.DATA_FILE_NAME_COLUMN, StringType))
```

Comment:
The value of Lineage used to be defined as non-nullable. But I reconsidered it and decided to make it nullable. This `false` check is no longer needed as the default value for nullable is true in StructField.

Reply:
I did it because in general a column is nullable unless there is a specific constraint to force it be non-nullable. I assumed we should not enforce a constraint unless there is a valid justification behind it.